.gitlab-ci.yml

```diff
@@ -11,8 +11,8 @@ variables:
   MENDER_ADDON_CONFIGURE_VERSION: latest
 
   ## Auto-update
-  RASPBIAN_URL: "https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2022-01-28/2022-01-28-raspios-bullseye-armhf-lite.zip"
-  RASPBIAN_NAME: 2022-01-28-raspios-bullseye-armhf-lite
+  RASPBIAN_URL: "https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2022-09-26/2022-09-22-raspios-bullseye-armhf-lite.img.xz"
+  RASPBIAN_NAME: 2022-09-22-raspios-bullseye-armhf-lite
```

Comment:
FYI, I have updated the image in #483.

Reply:
Great! Then I will close this one. It will reappear though, unless we disable it in https://github.com/mendersoftware/mender-convert/pull/490 (see [this comment](https://github.com/mendersoftware/mender-convert/pull/490#issuecomment-1275827908) in particular).